app/dashboard/helpers.py

```diff
@@ -557,6 +557,11 @@ def create_new_bounty(old_bounties, bounty_payload, bounty_details, bounty_id):
             logger.error(f'{e} encountered during new bounty creation for: {url}')
             new_bounty = None
 
+    if new_bounty.event:
+        for registration in HackathonRegistration.objects.filter(hackathon=new_bounty.event):
+            # TODO: Run this in background:
```

Comment:
Do we still need this todo / has the cronjob been set ?

Reply:
The cronjob has been set, but to test it needs working SendGrid and it does not work for me (despite I added my email to their "owner" emails list). Somebody test it.